packages/framework-core/src/services/graphql/graphql-generator.ts

```diff
@@ -218,6 +218,7 @@ function toCommandEnvelope(commandName: string, value: any, context: GraphQLReso
     requestID: context.requestID,
     currentUser: context.user,
     typeName: commandName,
+    responseHeaders: context.responseHeaders,
```

Comment:
I think that this is no longer needed, as we're keeping a reference in the graphQL dispatcher.

```suggestion
```